src/app/api/auth/login/route.ts

```diff
@@ -0,0 +1,80 @@
+import { NextResponse } from 'next/server';
+import type { NextRequest } from 'next/server';
+import type { User } from '@prisma/client';
+import { createAccessToken } from '@/app/lib/auth';
+import { AUTH_ACCESS_TOKEN } from '@/app/constants/auth';
+
+type RequestBody = {
+  token: string;
+};
+
+type GoogleUserInfo = {
+  email: string;
+  family_name: string;
+  given_name: string;
+  id: string;
+  locale: string;
+  name: string;
+  picture: string;
+  verified_email: boolean;
+};
+
+export const runtime = 'edge';
+const COOKIE_EXPIRE_IN_1_WEEK = 7;
+
+export async function POST(request: NextRequest) {
+  const body = await request.json();
+  const { token } = body as RequestBody;
+
+  if (!token) {
+    return NextResponse.json({ message: 'Token is required' }, { status: 400 });
+  }
+
+  const googleUserResponse = await fetch('https://www.googleapis.com/oauth2/v1/userinfo?alt=json', {
+    headers: {
+      Authorization: `Bearer ${token}`,
+    },
+  });
+  const googleUser = (await googleUserResponse.json()) as GoogleUserInfo;
```

Comment:
```suggestion
  const googleUser = await fetch('https://www.googleapis.com/oauth2/v1/userinfo?alt=json', {
    headers: {
      Authorization: `Bearer ${token}`,
    },
  }).then<GoogleUserInfo>(r => r.json());
```

Response 로 뭔가하는게 없다면 요렇게 한줄로 작성할수도있습니다

Reply:
@ysm-dev 
네넵 간결하고 좋은 것 같습니다!
[d3d88bb](https://github.com/Cyjin-jani/workoutManager/pull/11/commits/d3d88bb7de3a06e4ed7e020d132d85fbc1f77875)로 적용해보았습니다.
감사합니다🙌